test/tools/mongodb_reporter.js

```diff
@@ -0,0 +1,329 @@
+/* eslint-disable no-console */
+/* eslint-disable strict */
+/* eslint-disable @typescript-eslint/no-var-requires */
+//@ts-check
+
+'use strict';
+const mocha = require('mocha');
```

Comment:
This reporter is mainly to produce the xunit for evergreen which we aren't using here. Do we need the skipReason feature for a package as simple as this? We don't have this reporter outside the driver, and I don't feel as though its functionality has been missed.

Reply:
I think its less overhead to keep around things we don't use. It's also more overhead to have differences between packages. I would say my inclination for this package is that having this custom reporter provides less value than the cost of the difference between this repo and our driver.

It was copied over from libmongocrypt bindings that used to run in evergreen, I don't think we need to keep it in that package either.

Reply:
Okay, I'll remove.